src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -139,7 +144,11 @@ export default class GeoChartRenderer extends React.PureComponent<IGeoChartRende
     };
 
     private cleanupMap = (): void => {
-        this.chart.removeLayer(DEFAULT_DATA_SOURCE_NAME);
+        this.chart.removeLayer(DEFAULT_LAYER_NAME);
+        if (!this.geoData.size) {
```

Comment:
Please check null or set default value for `geoData` before accessing `size`
I'm just afraid that `componentDidUpdate` is called before `setupMap` for some unexpected cases

Reply:
fixed. thanks :)